lib/tasks/full_import.rake

```diff
@@ -1,33 +1,92 @@
-Import = Struct.new("Import", :import_class, :import_method, :folder)
-
-namespace :core do
-  desc "Import all data XMLs from legacy CORE"
-  task :full_import, %i[archive_path] => :environment do |_task, args|
-    archive_path = args[:archive_path]
-    raise "Usage: rake core:full_import['path/to/archive']" if archive_path.blank?
+namespace :import do
+  desc "Run a full import for the institutions listed in the named file on s3"
```

Comment:
is an institution a group of orgs?

Reply:
No, I'm just mixing up old core terminology because the exported file you get from old core has institutions in the name 🤦